makefile

```diff
@@ -1,5 +1,16 @@
+.PHONY: test test-component test-all init clean all
+
+test:
+	pipenv run pytest tests/unit
+
+test-component:
+	pipenv run pytest tests/component
+
+test-all:
+	pipenv run pytest tests
+
 init:
 	pipenv install --dev
 
-test:
-	pipenv run python -m unittest discover tests/
+clean:
+	rm -rf control.egg-info .pytest_cache */*.pyc */__pycache__
```

Comment:
Is this meant to clear cached byte code from a user's root directory?

Reply:
yeah the egg stuff is the setup.py aftermath (python packaging stuff), and the other stuff is to clear the cached bytecode if you want to. Not the most robust way to do it, the */*.pyc and */__pycache__ only work on a subdirectory one deep (I just tried it by making a bunch of em and a bunch of folders deep). Just saw this https://gist.github.com/lumengxi/0ae4645124cd4066f676 which looks more robust if we wanted to use it.